services/QuillLMS/engines/evidence/lib/evidence/evidence/prefilter_check.rb

```diff
@@ -0,0 +1,53 @@
+module Evidence
+  class PrefilterCheck
+    attr_accessor :prefilter_rules
+    MINIMUM_WORD_COUNT = 3
+    OPTIMAL_RULE_UID = '' # TODO: populate
+
+    PREFILTERS = {
+      # TODO: populate
+    }
+
+    def initialize(entry)
+      @entry = entry
+      @prefilter_rules = Evidence::Rule.where(rule_type: Evidence::Rule::TYPE_PREFILTER).includes(:feedbacks)
+      
+    end
+
+    def default_response
+      {
+        feedback: '',
+        feedback_type: Evidence::Rule::TYPE_PREFILTER,
+        optimal: true,
+        response_id: '',
```

Comment:
I think we're using "0" for this in some places in the code. Not a big deal because it isn't used for anything yet, but maybe we should standardize whether this is empty string or "0".

Reply:
SpellingCheck and Regex both use `''` 

Want to raise in the eng. meeting? I'm happy to change all instances then, + documentation, if that's what the consensus indicates.